themes/aframe/source/css/secondary.styl

```diff
@@ -168,6 +168,10 @@ a
     tr:not(:last-child) td
         border-bottom 1px solid #eee
 
+    td code
+        color $light
```

Comment:
per my suggestion [above](https://github.com/aframevr/aframe-site/pull/466/files#r143642796), I'd change this to `font-weight 600` too